### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#338] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324938-a3d89579-4c5c-4063-9f6e-127698b14368.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217810224-01703567-1046-4d55-b3e6-ae75b17d4154.png)

**Comment by simyKek** [comment]
C'è un errore nel 2° esercizio proposto da CiottoloMaggico: (q v -r) -> p dà nella parte a sinistra 1 e in quella a destra 1, perciò risulta come valore finale F = 1 (basta semplicemente analizzare i casi, senza scomporre, ma anche scomponendo viene 1).
